DatadogInternal/Tests/Codable/AnyCodableTests.swift

```diff
@@ -184,6 +184,7 @@ class AnyCodableTests: XCTestCase {
 
     func testGivenEncodableValueWrappedIntoCodableValue_whenEncoding_itProducesExpectedJSONRepresentation() throws {
         let encoder = JSONEncoder()
+        encoder.outputFormatting = [.sortedKeys]
```

Comment:
The order of keys in serialized JSON varies between iOS 17 and prior versions. We sort them, to get stable result.